tox.ini

```diff
@@ -7,7 +7,9 @@ deps=
     py27: mock
 commands=python -m unittest discover -v tests
 [flake8]
+ignore=E501
```

Comment:
Not sure we want to ignore lines too long? Perhaps we should change the max line length instead?

Reply:
OK, sure, what do you recommend?

Reply:
I just run `flake8`.

The changes also include reordering imports into blocks - first block containing standard lib. imports, second block containing external 3rd party imports, third block containing internal imports. Also, ensuring ascending order within blocks.

Nothing significant in terms of functionality, but it would be nice to keep imports clean and tidy.

Reply:
Cool you are also using flake8, I suggest to add this setup.cfg file then? https://stackoverflow.com/a/42325590/1515819

Reply:
About the reordering of the imports, I totally agree it's nicer, but there is good chances that the next developer will "forget" to order them the same way. Need to see if we can add a rule that checks this automatically.

Reply:
Actually use a `.flake8` file instead of `setup.cfg`, this way it will be obvious that we are using flake8 http://flake8.pycqa.org/en/2.6.0/config.html#per-project

Reply:
According to the documentation flake8 can be configured via the tox ini, so I'll just add the line length instruction and update the PR. I hope that's OK.

Reply:
A tool that parses modules to check for proper import block structure would be quite useful - who knows, perhaps someone out there has already written one!

Reply:
Yes looks like there is https://stackoverflow.com/a/58804315/1515819 but it involves a pre-commit as it's not available as a flake8 rule.

I'm gonna merge this PR for now. We can still add that later if we've got time

Thanks for that